### DISCUSSION TITLE
Radio button with label BELOW or ABOVE

### DISCUSSION THREAD

**Discussion by peppermintbunny** [quasarframework/quasar#16010] [Ideas / Proposals]
Please can you provide for a radio button with the label BELOW the radio button and for a radio button with the label ABOVE the radio button?  The label should have word wrap and the radio button should be centered above or below the label.  It would be fantastic if we could set the font size of the text.

Happy to provide more info if it will be helpful

Thank you.

**Comment by metalsadman** [comment]
you can wrap it in a QField

**Reply by peppermintbunny** [reply]
Thanks for your reply!!!!  But .... could you give me a snippet of code.  a radiobutton with the "button circle" on top and the text "A lot" underneath.  My javascript skills are under development ;-)
